pipelines/LSSTCam/ApPipe.yaml

```
@@ -0,0 +1,7 @@
+description: >-
+  AP pipeline specialized for LSSTCam Prompt Processing. This
+  file is imported in prompt_processing/pipelines/LSSTCam/ApPipe.yaml
+  which is used in realtime Prompt Processing production.
```

I realize this is a pre-existing issue, but I find both these descriptions a little problematic. The AP pipeline is simply the AP pipeline; while we do use it in PP, `ap_pipe` shouldn't assume or depend on that fact. Maybe:
```suggestion
  AP pipeline specialized for LSSTCam.
```